content/docs/docs-nav.yaml

```
@@ -66,34 +66,40 @@ nav:
+      - API docs:
+        - Custom renderers: '/docs/details/custom-api-docs-renderers/'
+        - Using grpc-docs: '/docs/details/grpc-docs/'
       - Configure GitHub Enterprise: '/docs/details/github-enterprise/'
       - GitHub app permissions: '/docs/details/github-app-permissions/'
-      - How Roadie connects: '/docs/details/how-roadie-connects/'
-      - Managing locations: '/docs/details/location-management/'
-      - Modelling dependencies: '/docs/details/showing-dependencies/'
-      - Previewing component changes: '/docs/details/previewing-changes/'
-      - Required annotations: '/docs/details/required-annotations/'
-      - Removing components: '/docs/details/unregister-components/'
-      - Requesting plugins: '/docs/details/requesting-plugins/'
-      - Setting secrets: '/docs/details/setting-secrets/'
-      - Troubleshooting the catalog: '/docs/details/troubleshooting-the-catalog/'
+      - Networking:
+        - Accessing AWS resources: '/docs/details/accessing-aws-resources/'
+        - Accessing public resources: '/docs/details/backend-reading-allow-list/'
+        - Allowlisting Roadie traffic: '/docs/details/allowlisting-roadie-traffic/'
+        - How Roadie connects: '/docs/details/how-roadie-connects/'
+      - The catalog:
+        - Managing locations: '/docs/details/location-management/'
+        - Modelling dependencies: '/docs/details/showing-dependencies/'
+        - Removing components: '/docs/details/unregister-components/'
+        - Previewing component changes: '/docs/details/previewing-changes/'
+        - Troubleshooting the catalog: '/docs/details/troubleshooting-the-catalog/'
+        - Validating catalog files: '/docs/details/validator/'
       - Testing page configurations: '/docs/details/sandbox-mode/'
-      - Using grpc-docs: '/docs/details/grpc-docs/'
+      - Working with plugins:
+        - Required annotations: '/docs/details/required-annotations/'
+        - Requesting plugins: '/docs/details/requesting-plugins/'
+      - Setting secrets: '/docs/details/setting-secrets/'
       - Updating the UI: '/docs/details/updating-the-ui/'
-      - Validating catalog files: '/docs/details/validator/'
-      - Writing scaffolder templates: '/docs/details/scaffolder/'
-  - Scaffolder-Cookbook:
-      - Create RFC: '/docs/scaffolder-cookbook/create-rfc/'
-      - Debug template: '/docs/scaffolder-cookbook/debug-template/'
-      - Append file: '/docs/scaffolder-cookbook/append/'
-      - Re-use Parameters: '/docs/scaffolder-cookbook/global-constants/'
-      - Edit Files: '/docs/scaffolder-cookbook/edit-file/'
+  - Scaffolder:
+      - Writing scaffolder templates: '/docs/scaffolder/writing-templates/'
+      - Recipies:
+        - Create RFC: '/docs/scaffolder/create-rfc/'
+        - Debug template: '/docs/scaffolder/debug-template/'
+        - Append file: '/docs/scaffolder/append/'
+        - Re-use Parameters: '/docs/scaffolder/global-constants/'
```

```suggestion
        - Re-use parameters: '/docs/scaffolder/global-constants/'
```

Everything else apart from brands/names are lowercase